lib/views.js

```diff
@@ -89,60 +25,36 @@ function configure(app) {
   }
   const paths = viewPaths.map(function(p) {return path.resolve(p);});
 
-  // add swig as the default template engine
-  app.engine('html', function(path, options, callback) {
-    let view;
-    try {
-      const swig = new swigcore.Swig({
-        autoescape: false,
-        cache: bedrock.config.views.cache ? 'memory' : false,
-        loader: swigLoaders.multipath({base: paths})
-      });
-      view = swig.compileFile(path)(options);
-    } catch(err) {
-      return callback(err);
-    }
-    callback(null, view);
-  });
+  // add the default template engine
+  // TODO: use configuration to allow other engines
+  app.engine('html', consolidate[bedrock.config.views.engine]);
   app.set('view engine', 'html');
   app.set('views', paths);
 
   if(bedrock.config.views.serviceUnavailable) {
     // service unavailable for all requests
-    app.all('*', function(req, res, next) {
+    app.all('*', function(req, res/*, next*/) {
```

Comment:
arrow function?

Reply:
I didn't arrowize all the things.  Should I?